samples/precompiled/Monitor.cs

```diff
@@ -11,7 +11,32 @@
 using Newtonsoft.Json.Linq;
 using Twilio;
 
-// To run this sample, you'll need a Weather Underground API key. Sign-up and documentation here: https://www.wunderground.com/weather/api/d/docs
+/* This sample demonstrates the Monitor workflow. In this pattern, the orchestrator function is
+ * used to periodically check something's status and take action as appropriate. While a
+ * Timer-triggered function can perform similar polling action, the Monitor has additional
+ * capabilities:
+ *
+ *   - manual termination (via request to the orchestrator termination endpoint)
+ *   - termination when some condition is met
+ *   - monitoring of multiple arbitrary subjects
+ *
+ * To run this sample, you'll need to define the following app settings:
+ *
+ *   - TwilioAccountSid: your Twilio account's SID
+ *   - TwilioAuthToken: your Twilio account's auth token
+ *   - TwilioPhoneNumber: an SMS-capable Twilio number
+ *   - WeatherUndergroundApiKey: a WeatherUnderground API key
+ *
+ * For Twilio trial accounts, you also need to verify the phone number in your MonitorRequest.
+ *
+ * Twilio: https://www.twilio.com
+ * WeatherUnderground API: https://www.wunderground.com/weather/api/d/docs
+ *
+ * View runtime logs:
+ *   - Locally at C:\Users\{user}\AppData\Local\Temp\LogFiles\Application\Functions
```

Comment:
nit: This location only applies for Windows machines. It will be different for macOS/Linux. We should probably clarify that so macOS/Linux customers don't feel left out.

Reply:
Moving this into a PR for the official docs so it's more visible/centralized.